.editorconfig

```diff
@@ -0,0 +1,2 @@
+indent_style = space
+indent_size = 2
```

Comment:
I think I saw this last time too - please change to 4, simply because that's the community standard (and what existing files would be indented to)

Always choose the community standard for style - it enables easier access & contribution by others.

Reply:
Thanks for the hint. Adjusted.